WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -105,6 +105,11 @@
     public static final String WORDPRESS_PURCHASES_URL = "https://wordpress.com/purchases";
 
     /**
+     * url for redirecting free users to empty their sites (start over)
+     */
+    private final String WORDRESS_EMPTY_SITE_SUPPORT_URL = "https://en.support.wordpress.com/empty-site/";
```

Comment:
I changed this to `public static` for consistency and to avoid that AS warning. There was also a typo in the name, fixed that as well in 14ce6183649cee51b32d6d1620ec6df6f82d0bed